dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/activations/Activations.scala

```diff
@@ -0,0 +1,10 @@
+package me.shadaj.scalapy.tensorflow.api.keras.activations
+
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonEnum
+
+enum Activation(override private[api] val v: String) extends PythonEnum(v){
+  case Elu   extends Activation("elu")
```

Comment:
I'd rather use `ELU`/`RELU` uppercase as Scala identifiers

Reply:
Or even expand the abbrevs... up to you

Reply:
I haven't even had full names in my lectures at uni it would be confusing as hell.

Reply:
if we want real names it's `ELU` and `ReLU`